debian/stretch/foreman/control

```diff
@@ -3,7 +3,7 @@ Maintainer: Greg Sutcliffe <greg.sutcliffe@gmail.com>
 Section: web
 Priority: extra
 Standards-Version: 3.9.1
-Build-Depends: debhelper (>= 7), cdbs, bundler (>= 1.3), ruby2.3-dev, git | git-core,
+Build-Depends: debhelper (>= 9), bundler (>= 1.3), ruby2.3-dev, git | git-core,
```

Comment:
Now you edited too much ;)

On stretch we need `debhelper (>= 9.20160709)` as we don't need `dh-systemd` anymore ;)

Reply:
oops!